_posts/newsletter/2021-03-11-this-month-in-solid.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: newsletter
+title: This Month in Solid 2020-03-11
+permalink: /newsletter/2020-03-11
```

Comment:
Whoops, wrong year again! I'll fix this right away as I don't think anyone would disagree with that ;)

```suggestion
title: This Month in Solid 2021-03-11
permalink: /newsletter/2021-03-11
```